ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ParkingZoneApp.Models;
+using ParkingZoneApp.Services.Interfaces;
+using ParkingZoneApp.ViewModels.ParkingSlots;
+
+namespace ParkingZoneApp.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize]
+    public class ParkingSlotController : Controller
+    {
+        private readonly IParkingSlotService _parkingSlotService;
+        public ParkingSlotController(IParkingSlotService parkingSlotService)
+        {
+            _parkingSlotService = parkingSlotService;
+        }
+
+        public IActionResult Index()
+        {
+            var parkingSlots = _parkingSlotService.GetAllParkingSlots();
```

Comment:
here you are taking all parking slots. we should not do like that. every parking zone has slots we should take parking slots belongs to specific parking zone. in parking zone index you should add button called "Slots" and should return all slots belongs to parking zone